### PR TITLE
Added rules page

### PR DESCRIPTION
**Pull request by anupkgurung**
Added rules page with rule listing

### CODE REVIEW

File: src/page/Rules/rules.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.ul-list-flex {
+    display: flex;
+    flex-direction: column;    
+    align-items: flex-start;
+}
+.rule-container {
+    display : flex;
+    flex-direction : column;
+    justify-content : center;
+    align-content : center;
```
Comment (Aditi0505) [inline src/page/Rules/rules.css]: Since you are not using flex-wrap here, align-content won't work. Any specific reason to use this?

**Review by Aditi0505 — APPROVED** [review]
Suggestions:
- Add description
- Add Screenshots
- Add preview links
- In folder structure, try to maintain consistency with cases. Eg: Rules.jsx and Rules.css  ( follow Title case here instead of rules.css)

- Rest looks good and Easily readable code

File: src/page/Rules/rules.css (hunk 1 above)
Comment (anupkgurung) [reply src/page/Rules/rules.css]: i had removed justify and align content instead added align-items.
forgot to push. thanks

File: src/page/Rules/Rules.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+];
+export const Rules = () => {
+    return (
+        <>
+            <div className="m-2 pd-4 rule-container">
+                <div className="m-1 pd-2 rules-card">
+                    <img src="https://raw.githubusercontent.com/anupkgurung/images/main/quizimg/rulespage.png" alt="rules" />
+                    <ul className="ul-list-flex">
+                    {rules.map(item => (
+                        <li className="pd-2">{item}</li>
```
Comment (ishitakeshawani) [inline src/page/Rules/Rules.jsx]: add key here.

**Review by ishitakeshawani — APPROVED** [review]
Readable code.
Just put a link to preview.

**Pull request merged**